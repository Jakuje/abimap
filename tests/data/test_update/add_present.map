# This map file was created with symbol_version.py

ADD_PRESENT_1_0_0
{
    global:
        symbol;
    local:
        *;
} ;

