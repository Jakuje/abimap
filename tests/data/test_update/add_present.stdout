Added:
    symbol

# This map file was automatically updated

ADD_PRESENT_1_1_0
{
    global:
        symbol;
} ADD_PRESENT_1_0_0;

ADD_PRESENT_1_0_0
{
    global:
        symbol;
    local:
        *;
} ;

